src/MigrationTools.Host/MigrationToolHost.cs

```diff
@@ -66,7 +67,18 @@ public static IHostBuilder CreateDefaultBuilder(string[] args)
                  services.AddOptions();
                  // Sieralog
                  services.AddSingleton<LoggingLevelSwitch>(levelSwitch);
+
+                 // Application Insights
+
+                 Microsoft.ApplicationInsights.WorkerService.ApplicationInsightsServiceOptions aiOptions
+                    = new Microsoft.ApplicationInsights.WorkerService.ApplicationInsightsServiceOptions
+                    {
+                        // Disables adaptive sampling.
```

Comment:
does the workservice type disable adaptive sampling, or some config?

Reply:
Looks to be just a left over comment.

Reply:
I have removed it in the next pull